mcp/sensors/acs.c

```diff
@@ -44,6 +44,17 @@
 #include "gps.h"
 #include "xsc_network.h"
 
+
+/**
+ * @brief 64 various gyro matrices packaged together with a gyro mask to index them
+ * sub structs:
+ * row 0: roll conversions
+ * row 1: yaw conversions
+ * row 2: el conversions
+ * col 0 + 1: gyro 1 and 2 roll
+ * col 2 + 3: gyro 1 and 2 yaw
+ * col 4 + 5: gyro 1 and 2 el
+ */
 static const float gy_inv[64][3][6] =
```

Comment:
can this live in the header file instead?

Reply:
idk, but I am not going to make structural changes without a strong reason.